trade-module/src/main/java/io/edpn/backend/trade/adapter/persistence/entity/mapper/MybatisMarketDatumEntityMapper.java

```diff
@@ -0,0 +1,50 @@
+package io.edpn.backend.trade.adapter.persistence.entity.mapper;
+
+import io.edpn.backend.trade.adapter.persistence.entity.MybatisCommodityEntity;
+import io.edpn.backend.trade.adapter.persistence.entity.MybatisMarketDatumEntity;
+import io.edpn.backend.trade.application.domain.MarketDatum;
+import io.edpn.backend.trade.application.dto.persistence.entity.MarketDatumEntity;
+import io.edpn.backend.trade.application.dto.persistence.entity.mapper.CommodityEntityMapper;
+import io.edpn.backend.trade.application.dto.persistence.entity.mapper.MarketDatumEntityMapper;
+import lombok.RequiredArgsConstructor;
+
+@RequiredArgsConstructor
+public class MybatisMarketDatumEntityMapper implements MarketDatumEntityMapper<MybatisMarketDatumEntity> {
+    
+    private final CommodityEntityMapper<MybatisCommodityEntity> commodityEntityMapper;
+    
+    @Override
+    public MarketDatum map(MarketDatumEntity marketDatumEntity) {
+        return new MarketDatum(
+                //TODO: Remap to validatedCommodity?
```

Comment:
What's the difference between validatedCommodiity and MarketDatum?

Reply:
> What's the difference between validatedCommodiity and MarketDatum?

IIRC, validated commodities contain only whitelisted commodities.

Reply:
commodity ingested from EDDN
ValidatedCommodity a scrubbed and enriched object we export.

MarketDatum wholly different and true to its name is a commodity + station + timestamp object